src/NServiceBus.Gateway/Recoverability/DefaultRetryPolicy.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus.Gateway
+{
+    using NServiceBus.Transports;
+    using System;
+
+    class DefaultRetryPolicy
+    {
+        public static Func<IncomingMessage, Exception, int, TimeSpan> BuildWithDefaults()
+        {
+            return Build(4, TimeSpan.FromSeconds(60));
+        }
+
+        public static Func<IncomingMessage, Exception, int, TimeSpan> Build(int numberOfRetries, TimeSpan timeIncrease)
+        {
+            return (message, exception, currentRetry) => currentRetry > numberOfRetries
```

Comment:
it seems to me like thi should be `currentRetry >= numberOfRetries`.
e.g. the default policy configures for 4 retries, but it will actually do 3 retries as stated by the acceptance test.